### YamlMime:JavaType
uid: "com.azure.communication.identity.models.IdentityError"
fullName: "com.azure.communication.identity.models.IdentityError"
name: "IdentityError"
nameWithType: "IdentityError"
summary: "The Phone Number Services error."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "java.lang.Object.clone()"
- "java.lang.Object.equals(java.lang.Object)"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.hashCode()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.toString()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class IdentityError"
constructors:
- "com.azure.communication.identity.models.IdentityError.IdentityError(java.lang.String,java.lang.String,java.lang.String,java.util.List<com.azure.communication.identity.models.IdentityError>)"
methods:
- "com.azure.communication.identity.models.IdentityError.getCode()"
- "com.azure.communication.identity.models.IdentityError.getDetails()"
- "com.azure.communication.identity.models.IdentityError.getMessage()"
- "com.azure.communication.identity.models.IdentityError.getTarget()"
type: "class"
metadata: {}
package: "com.azure.communication.identity.models"
artifact: com.azure:azure-communication-identity:1.1.7
